### PR TITLE
README: update README.md

### PR DESCRIPTION
**Pull request by overvenus**
PTAL @siddontang @BusyJay

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by BusyJay** [comment]
Resolve conflicts first.

**Comment by overvenus** [comment]
@BusyJay Here we go.

File: README.md [hunk 1]
```diff
@@ -20,19 +18,25 @@ This library has been tested against RocksDB 3.13.1 on linux and OSX.  The 0.3.0
   - [ ] slicetransform
```
Comment (BusyJay) [inline README.md]: Prefix seek and slicetransform should have been implemented. @zhangjinpeng1987

Comment (zhangjinpeng1987) [inline README.md]: yes

Comment (zhangjinpeng1987) [inline README.md]: @overvenus  memtable_insert_with_hint also has supported.

Comment (zhangjinpeng1987) [inline README.md]: @overvenus Should we list all features we supported?

Comment (overvenus) [inline README.md]: Sure, I will check it out.

**Comment by overvenus** [comment]
PTAL @zhangjinpeng1987 @BusyJay @siddontang

File: README.md [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
-Feedback and pull requests welcome!  If a particular feature of RocksDB is important to you, please let me know by opening an issue, and I'll prioritize it.
+Feedback and pull requests welcome! If a particular feature of RocksDB is important to you, please let us know by opening an issue, and we will prioritize it.
 
 ###### Prerequisite: RocksDB
 
 First, use your system's package manager to install snappy.  This is optional, but lets rocksdb take advantage of better compression, and some code may require it.
 
 To install rocksdb, please refer to it's installation guide. For Windows users, please make sure you configure rocksdb with `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` enabled.
 
+Or enable feature `static-link`, the crate will download and complie RocksDB automatically, including it's dependencies.
```
Comment (hhkbp2) [inline README.md]: s/it's/its/

**Comment by hhkbp2** [comment]
Rest LGTM

**Pull request merged**